packages/vite-plugin-checker/package.json

```diff
@@ -40,14 +40,14 @@
   "homepage": "https://github.com/fi3ework/vite-plugin-checker",
   "dependencies": {
     "@babel/code-frame": "^7.12.13",
+    "@volar/typescript": "^2.3.0",
```

Comment:
q: is there a reason why `@volar/typescript` is now added to `dependencies`?

```
+-- vite-plugin-checker@0.7.0
| `-- @volar/typescript@2.3.4
|   `-- @volar/language-core@2.3.4
`-- vue-tsc@2.0.24
  +-- @volar/typescript@2.4.0-alpha.5
  | `-- @volar/language-core@2.4.0-alpha.5
  `-- @vue/language-core@2.0.24
    `-- @volar/language-core@2.4.0-alpha.5
```

Reply:
Just realized it can use the same approach to import `@volar/typescript` from `vue-tsc`. I'll fix it.

Reply:
#333